I checked out to your branch, provisioned the VM again (vagrant provision), logged in the VM (vagrant ssh), ran the tests and I got the error bellow:

Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/local/lib/python2.7/dist-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/vagrant/test/test_models.py", line 6, in <module>
    from mock import MagicMock
ImportError: No module named mock

I guess you need to add this new dependency (mock) in chef's recipe. Fix it in #24 should be ok.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,61 @@
+language: python
+python:
+  - "2.7"
+
+services:
+  - postgresql
+
+addons:
+  hosts:
+    - bmdb1.debian.org
+
+env:
+  - TPORT=5435
+
+virtualenv:
+  system_site_packages: true
+
+before_install:
+  - sudo apt-get update -qq
+
+install:
+  ### postgres packages
+  - sudo apt-get install postgresql-9.1 postgresql-9.1-debversion
+    postgresql-contrib-9.1
+  ### python packages
+  - sudo apt-get install -qq python-argparse python-debian python-apt
+    python-debianbts python-inotifyx python-paste python-psycopg2 python-pyramid
+    python-sqlalchemy python-subversion python-pip wget
+  ### pip packages
+  - sudo pip install coveralls
+  - sudo pip install flake8 # ubuntu 12 does not have python-flake8 in its repo
+
+before_script:
+  ### postgresql.conf
+  - sudo bash -c "echo 'port = 5435' >> /etc/postgresql/9.1/main/postgresql.conf"
+  ### pg_hba.conf
+  - sudo sed -i -e "1ihost pet pet 127.0.0.1/32 trust" /etc/postgresql/9.1/main/pg_hba.conf
+  ### useradd / restart
+  - sudo useradd pet
+  - sudo service postgresql restart
+  ### create user/db
+  - sudo su postgres -c "createuser -p $TPORT-s -r -d pet &>/dev/null"
+  - sudo su postgres -c "createdb -p $TPORT -O pet pet &>/dev/null"
+  ### debversion.sql
+  - sudo su postgres -c "psql -p $TPORT pet < /usr/share/postgresql/9.1/extension/debversion--1.0.8.sql"
+  ### pet insert
+  - sudo su pet -c "./pet-update -c -nc"
+  - sudo su pet -c "psql -p $TPORT pet --command \"INSERT INTO team (name, maintainer, url) VALUES ('pkg-perl', 'Debian Perl Group <pkg-perl-maintainers@lists.alioth.debian.org>', 'http://pkg-perl.alioth.debian.org/');\""
+  - sudo su pet -c "psql -p $TPORT pet --command \"INSERT INTO repository (name, type, root, web_root, team_id) VALUES ('git','git','https://pet.alioth.debian.org/pet2-data/pkg-perl/git-pkg-perl-packages.json','http://anonscm.debian.org/gitweb/?p=pkg-perl/packages', 1);\""
+
+script:
+  - flake8 . --ignore E121,E501,F401,E712,E711,E123,E241
+  - nosetests --with-coverage --cover-package=pet
+
+after_success: coveralls
+
+branches:
+  only:
+    - master
+notifications:
+  email: false

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
-[![CoverageStatus](https://coveralls.io/repos/github/PET-UnB/pet/badge.svg?branch=travis)](https://coveralls.io/github/PET-UnB/pet?branch=travis)
-[![BuildStatus](https://travis-ci.org/PET-UnB/pet.svg?branch=travis)](https://travis-ci.org/PET-UnB/pet.svg?branch=travis)
+[![CoverageStatus](https://coveralls.io/repos/github/PET-UnB/pet/badge.svg?branch=master)](https://coveralls.io/github/PET-UnB/pet?branch=master)
+[![BuildStatus](https://travis-ci.org/PET-UnB/pet.svg?branch=master)](https://travis-ci.org/PET-UnB/pet.svg?branch=master)
 
 # PET - Package Entropy Tracker
 

--- a/cookbooks/pet/recipes/default.rb
+++ b/cookbooks/pet/recipes/default.rb
@@ -19,6 +19,7 @@ package 'python-nose'
 
 #installing dependency that has no package on debian
 execute 'pip install pyramid_chameleon'
+execute 'pip install coveralls'
 
 #creating user pet
 user 'pet' do

--- a/test/test_bts.py
+++ b/test/test_bts.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.bts
+
+
+class TestPetBts(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_classifier.py
+++ b/test/test_classifier.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.classifier
+
+
+class TestPetClassifier(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_exceptions.py
+++ b/test/test_exceptions.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.exceptions
+
+
+class TestPetExceptions(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_init.py
+++ b/test/test_init.py
@@ -1,0 +1,11 @@
+import unittest
+import pet
+
+
+class TestPetInit(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_engine(self):
+        pass

--- a/test/test_perlre.py
+++ b/test/test_perlre.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.perlre
+
+
+class TestPetPerlre(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_sql.py
+++ b/test/test_sql.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.sql
+
+
+class TestPetSql(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_update.py
+++ b/test/test_update.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.update
+
+
+class TestPetUpdate(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_vcs.py
+++ b/test/test_vcs.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.vcs
+
+
+class TestPetVcs(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

--- a/test/test_watch.py
+++ b/test/test_watch.py
@@ -1,0 +1,11 @@
+import unittest
+import pet.watch
+
+
+class TestPetWatch(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_something(self):
+        pass

